### DISCUSSION TITLE
Add `npx create-subgraph@latest` command

### DISCUSSION THREAD

**Discussion by schmidsi** [graphprotocol/graph-tooling#1636] [Ideas]
Inspired by [Scaffold-ETH 2](https://scaffoldeth.io/), I think it would be cool to have a `npx create-subgraph@latest` command. I'm also open to `npx @graphprotocol/create-subgraph`. It has the following advantages:

- Builders are not forced to install the Graph CLI locally as a first step
- Running that command ensures, that it will always install the latest version of the dependencies. With Graph CLI locally in installed, I have to check manually if there is a newer version available.

I think this would not be super hard to do and could be a "Good first issue". Thoughts?

**Comment by YaroShkvorets** [comment]
Tracked here: https://github.com/graphprotocol/graph-tooling/issues/1467
